pjmedia/src/pjmedia/transport_srtp_dtls.c

```diff
@@ -1361,6 +1361,52 @@ static pj_status_t dtls_on_recv(pjmedia_transport *tp, unsigned idx,
         (ds->setup == DTLS_SETUP_ACTPASS || ds->setup == DTLS_SETUP_PASSIVE))
     {
         pj_status_t status;
+
+#if defined(PJMEDIA_SRTP_DTLS_CHECK_HELLO_ADDR) && PJMEDIA_SRTP_DTLS_CHECK_HELLO_ADDR!=0
+        pjmedia_transport_info info;
+        pj_sockaddr src_addr, rem_addr;
+        pj_bool_t rem_addr_avail = PJ_TRUE;
+        pjmedia_transport_get_info(ds->srtp->member_tp, &info);
+        if (idx == RTP_CHANNEL) {
+            pj_sockaddr_cp(&src_addr, &info.src_rtp_name);
+            if (!pj_sockaddr_has_addr(&ds->rem_addr)) {
+                rem_addr_avail = PJ_FALSE;
+            } else {
+                pj_sockaddr_cp(&rem_addr, &ds->rem_addr);
+            }
+        } else {
+            pj_sockaddr_cp(&src_addr, &info.src_rtcp_name);
+            if (!pj_sockaddr_has_addr(&ds->rem_rtcp)) {
+                rem_addr_avail = PJ_FALSE;
+            } else {
+                pj_sockaddr_cp(&rem_addr, &ds->rem_rtcp);
+            }
+        }
+        if (!rem_addr_avail) {
+            char addr[PJ_INET6_ADDRSTRLEN];
+
+            PJ_LOG(2, (ds->base.name, "DTLS-SRTP %s ignoring %lu bytes "
+                "from [%s], remote address not available",
+                CHANNEL_TO_STRING(idx), (unsigned long)size,
+                pj_sockaddr_print(&src_addr, addr, sizeof(addr), 3)));
+
+            DTLS_UNLOCK(ds);
+            return PJ_EIGNORED;
```

Comment:
Perhaps this should be PJ_SUCCESS instead? As PJ_EIGNORED means leaving the packet to be processed by the parent transport (SRTP).